tests/foreman/cli/test_satellitesync.py

```diff
@@ -1480,13 +1693,15 @@ def test_positive_import_content_for_disconnected_sat_with_existing_content(
             1. Product with synced custom repository, published in a CV.
 
         :steps:
-            1. Run complete export of the CV.
-            2. On Disconnected satellite, create a cv with same name as cv on 2 and with
-               'import-only' selected.
-            3. Run the import command.
+            1. Run complete export of the CV from setup.
+            2. On Disconnected satellite, create a CV with the same name as setup CV and with
+               'import-only' set to False and run the import command.
+            3. On Disconnected satellite, create a CV with the same name as setup CV and with
+               'import-only' set to True and run the import command.
```

Comment:
Shouldnt this be two different cases or parameterized ? Becasue failing step 2 will skip skip step 3 assertion.

Reply:
Well, it could. On the other hand here we leverage a single export to check the import behaves as expected in both cases, so we follow the "do more with less" strategy. To me it feels like... accurate as is.